.github/workflows/create-release.yaml

```diff
@@ -1,24 +1,45 @@
 ---
-name: Create Release
+name: Create release
+
 on:
   workflow_dispatch:
     inputs:
-      input_var:
-        description: "Say Hi!"
-        required: true
+      release_tag:
+        description: "Release Tag:"
+        type: string
+      target_branch:
+        description: "Target Branch:"
+        type: string        
   workflow_call:
     inputs:
-      input_var:
+      release_tag:
+        description: "Release Tag:"
         type: string
-        required: true
+      target_branch:
+        description: "Target Branch:"
+        type: string  
 
 env:
-  INPUT_VAR: ${{ inputs.input_var }}
+  RELEASE_TAG: ${{ inputs.release_tag }}
+  TARGET_BRANCH: ${{ inputs.target_branch }}
 
-jobs:
-  say-hello:
-    runs-on: ubuntu-latest
+permissions:
+  contents: write
 
+jobs:
+  release:
+    name: Create opendatahub-io/kubeflow release
+    runs-on: ubuntu-22.04
     steps:
-      - name: Say Hello
-        run: echo "${{ env.INPUT_VAR }}, World!"
+      - name: Check out the repository to the runner
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Invoke script to handle creating release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why two github tokens?

Reply:
I had originally gotten an error around the lack of `GH_TOKEN` breaking something... let me double-check and I'll update this comment shortly...

Reply:
OK, I can confirm code functions appropriately w/out `GH_TOKEN` set (but with preserving `GITHUB_TOKEN`)

This comment does a nice job explaining the history of these 2 vars as it pertains to GHA
- https://github.com/cli/cli/issues/2922#issuecomment-1886614283

I can only hypothesize I was initially not setting **ANY** `env` values.. and the `gh` CLI reported the error around `GH_TOKEN` - so I then (mistakenly) added them both...

Only `GITHUB_TOKEN` need be supplied ✅